Backend/ServiceLayer/Services/LoggingService.cs

```diff
@@ -82,5 +84,43 @@ private FormUrlEncodedContent getLogContent(LogRequestDTO newLog, string signatu
 
             return content;
         }
+
+        public bool notifyAdmin(System.Net.HttpStatusCode notify, FormUrlEncodedContent content)
+        {
+            if (notify != System.Net.HttpStatusCode.OK)
+            {
+                var fromAddress = new MailAddress("nightwatch491@gmail.com", "Night Watch");

```

Comment:
This type of stuff should come from environmental variables

Reply:
Theres an example of how to get an environmental variable for the db context class

Reply:
Why should this be an environmental variable, I understand if this was a global constant but why environmental?